src/styles/blocks/catalog.scss

```
@@ -1,13 +1,9 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-  align-self: center;
+  display: grid;
+  grid-template-columns: repeat(auto-fill, 200px);

```

можно задать количество колонок как в задании и тогда не нужен width:
```suggestion
  grid-template-columns: repeat(4, 200px);

```